plugins/ipam/commands/commands.go

```diff
@@ -100,17 +100,28 @@ func Del(args *skel.CmdArgs) error {
 }
 
 func del(ipManager ipstore.IPAllocator, ipamConf *config.IPAMConfig) error {
-	if ipamConf.IPV4Address.IP == nil {
-		return errors.New("del commands: ip address is required for deletion")
-	}
+	var releasedIP string
 
-	err := ipManager.Release(ipamConf.IPV4Address.IP.String())
-	if err != nil {
-		return err
+	if ipamConf.IPV4Address.IP != nil {
```

Comment:
I think `ipamConf.IPV4Address.IP != nil && ipamConf.IPV4Address.IP.String() == "<nil>"` is a stronger check since this is [not a pointer type](https://github.com/richardpen/amazon-ecs-cni-plugins-1/blob/eae4a48d251e6481aedaa7ec9f6564efdbad5f33/plugins/ipam/config/config.go#L40).